packages/language/src/preprocessor/instruction-interpreter.ts

```diff
@@ -1385,13 +1383,15 @@ function parseAndEvaluateProcedure(
       context,
     );
   }
-
   const localContext = createLocalContext(context, procedure);
   let value = runProcedure(procedure, evaluatedArgs, localContext);
-  if (isScalarValue(value) && suffix) {
+  if (isScalarValue(value) && immediateFollow && tokens[index + advance]) {
     // Add the suffix to the value, as it was immediately following the procedure call
```

Comment:
The documentation talks about a suffix, which is not present anymore in the current version. This can be misleading.